<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniDraw</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFigure">
    <property name="title">
     <string>Figure</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionFree"/>
    <addaction name="actionText"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClear"/>
    <addaction name="actionEraser"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuPenStyle">
     <property name="title">
      <string>PenStyle</string>
     </property>
     <addaction name="actionNoPen"/>
     <addaction name="actionSolidLine"/>
     <addaction name="actionDashLine"/>
     <addaction name="actionDotLine"/>
     <addaction name="actionDashDotLine"/>
     <addaction name="actionDashDotDotLine"/>
    </widget>
    <widget class="QMenu" name="menuPenWidth">
     <property name="title">
      <string>PenWidth</string>
     </property>
     <addaction name="action1px"/>
     <addaction name="action2px"/>
     <addaction name="action3px"/>
     <addaction name="actionCustom"/>
    </widget>
    <widget class="QMenu" name="menuBrushStyle">
     <property name="title">
      <string>BrushStyle</string>
     </property>
     <addaction name="actionSolid"/>
     <addaction name="actionDense1"/>
     <addaction name="actionDense2"/>
     <addaction name="actionDense3"/>
     <addaction name="actionDense4"/>
     <addaction name="actionDense5"/>
     <addaction name="actionDense6"/>
     <addaction name="actionDense7"/>
     <addaction name="actionHorizontalLine"/>
     <addaction name="actionVerticalLine"/>
     <addaction name="actionCrossLine"/>
     <addaction name="actionBDiagLine"/>
     <addaction name="actionFDiagLine"/>
     <addaction name="actionDiagCrossLine"/>
    </widget>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
    <addaction name="menuPenStyle"/>
    <addaction name="menuPenWidth"/>
    <addaction name="actionBrushColor"/>
    <addaction name="menuBrushStyle"/>
    <addaction name="actionHollow"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Me"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuSave"/>
   <addaction name="menuFigure"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_As"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionFree"/>
   <addaction name="actionText"/>
   <addaction name="actionEraser"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionBrushColor"/>
   <addaction name="actionHollow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionFree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Free</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionNoPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NoPen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSolidLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SolidLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionDashLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DotLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionDashDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashDotLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionDashDotDotLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DashDotDotLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionAbout_Me">
   <property name="text">
    <string>About Me</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action1px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1px</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="action2px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2px</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="action3px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3px</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBrushColor">
   <property name="text">
    <string>BrushColor</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionHollow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hollow</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionDense1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense1</string>
   </property>
  </action>
  <action name="actionDense2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense2</string>
   </property>
  </action>
  <action name="actionDense3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense3</string>
   </property>
  </action>
  <action name="actionDense4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense4</string>
   </property>
  </action>
  <action name="actionDense5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense5</string>
   </property>
  </action>
  <action name="actionDense6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense6</string>
   </property>
  </action>
  <action name="actionDense7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dense7</string>
   </property>
  </action>
  <action name="actionHorizontalLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HorizontalLine</string>
   </property>
  </action>
  <action name="actionVerticalLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>VerticalLine</string>
   </property>
  </action>
  <action name="actionCrossLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CrossLine</string>
   </property>
  </action>
  <action name="actionBDiagLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BDiagLine</string>
   </property>
   <property name="toolTip">
    <string>BDiagLine</string>
   </property>
  </action>
  <action name="actionFDiagLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FDiagLine</string>
   </property>
   <property name="toolTip">
    <string>FDiagLine</string>
   </property>
  </action>
  <action name="actionDiagCrossLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DiagCrossLine</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
